CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # What's New?
 
-## 1.13
-Improvements:
```

Comment:
Don't delete the 1.13 section.  This will make it more work to cherry pick to the release branch I created.  Just add the section underneath.

Reply:
Plus we'd need to add it back in when you're done.